C/README.md

```diff
@@ -1,3 +1,75 @@
+
```

Comment:
You only need to add your program's hyperlink in the README.md file. Check out other PRs for reference.